cinema/serializers.py

```diff
@@ -1 +1,99 @@
-# write serializers here
+from rest_framework import serializers
+
+from cinema.models import (
+    Genre,
+    Actor,
+    CinemaHall,
+    Movie,
+    MovieSession,
+)
+
+
+class GenreSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = Genre
+        fields = ("id", "name")
+
+
+class ActorSerializer(serializers.ModelSerializer):
+    full_name = serializers.CharField(source="__str__", read_only=True)
+
+    class Meta:
+        model = Actor
+        fields = ("id", "first_name", "last_name", "full_name")
+
+
+class CinemaHallSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = CinemaHall
+        fields = ("id", "name", "rows", "seats_in_row", "capacity")
```

Comment:
The `capacity` field is included in the `fields` tuple, but it is not defined in the `CinemaHall` model. If `capacity` is a calculated field, it should be defined in the serializer. Otherwise, it should be removed from the `fields` tuple.